docs/BR.md

```diff
@@ -390,6 +390,8 @@ The script outputs:
 
 **Root Certificate**: The self-signed Certificate issued by the Root CA to identify itself and to facilitate verification of Certificates issued to its Subordinate CAs.
 
+**Server Certificate**: An X.509v3 Subscriber Certificate that does not contain a `basicConstraints` extension with the `cA` field set to true.
```

Comment:
I found this term to be confusing, as I think the commonly understood meaning of "Server Certificate" is a certificate that can be used for TLS server authentication regardless of whether it is end-entity or a CA certificate. In other words, a "Server Certificate" is a certificate that expresses the serverAuth EKU (or lack of any EKU, etc). Case in point: the Server Certificate WG concerns itself with more than just end-entity certificates.

Any reason why we can't use the term "End-Entity" or "Leaf" Certificate?

Reply:
I'm not sure I fully buy that argument; a server certificate is one that can be used to authenticate a server (i.e. id-kp-serverAuth *and* a subjectAltName extension). The SCWG concerns itself with the issuance of server certificates, which thus includes policies around CA certificates used to (transitively) issue server certificates.

I'm also not 100% convinced on this language myself, and a broader cleanup is necessary, since "Subscriber" literally means "everyone who has a cert" (as it also includes the Terms of Use, which is what the CA and its affiliates execute). But we need a better term, and one that is precise and unambiguous in how to assess (most commonly, the basicConstraints is the property we're looking at, and yes, that's ignoring X509v1)

Reply:
I agree that we should revise both of these definitions. Especially because this should be an authoritative source on issues like these.

Reply:
As the term "Subscriber Certificate" is used in various places to describe requirements for end-entity certificates, I would argue that (should we take this approach), the new definition should be for Subscriber Certificate means end entity - even if that leads to a somewhat inconsistent use of the term Subscriber.

Reply:
@wthayer From when I looked through, it’s about half and half for things that would need updating.

However, this does seem to argue for a split along what I outlined above - avoiding the new term and splitting the general profile from the Subscriber cert profile. I’ll update this PR to reflect that

Reply:
@sleevi, I like your suggestion to borrow language from section 8.1. I don't think the downside you mentioned regarding X509v1 certificates is an issue here, as no CA could embed a CABF Policy OID in a non-X509v3 certificate as doing so would run afoul of section 7.1.1 (and thus wouldn't be issuing a Certificate in compliance with the BRs).